test/battlerTags/stockpiling.test.ts

```diff
@@ -149,12 +149,8 @@ describe("BattlerTag - StockpilingTag", () => {
       subject.onOverlap(mockPokemon);
       expect(subject.stockpiledCount).toBe(3);
 
-      vi.spyOn(game.scene, "unshiftPhase").mockImplementationOnce((_phase) => {
-        throw new Error("Should not be called a fourth time");
-      });
-
       // fourth stack should not be applied
-      subject.onOverlap(mockPokemon);
+      // subject.onOverlap(mockPokemon);
```

Comment:
```suggestion
      subject.onOverlap(mockPokemon);
```
(The test is testing that calling `onOverlap()` a fourth time does not apply a fourth layer of the Stockpiling tag)

Reply:
Oh whops, this must have slipped through while debugging. Thanks